pages/in/register/index.vue

```diff
@@ -191,6 +199,11 @@ export default {
     },
   },
   async mounted() {
+    const { redirect } = this.$route.query;
+    if (redirect && isValidRedirectUrl(redirect)) {
```

Comment:
Also we should not need to manually call `decodeURIComponent` if we get `redirect` from `this.$route.query`, please confirm

Reply:
👌 Vue Router has already decoded the URL
[🎨 Remove decodeURIComponent](https://github.com/likecoin/like-co/pull/1742/commits/336037823fa3cb081a21cb1d5b8f78a7f5c99a15)

Reply:
Do we still need decode in parseRedirectUrl ?
https://github.com/likecoin/like-co/pull/1742/files#diff-b41ab5cb5e0ab92f72bc78e948a8acfbde2d60d68ce1831b7089bdd9c4bf6739R428

Reply:
[🎨 Remove decodeURIComponent](https://github.com/likecoin/like-co/pull/1742/commits/5cefd3ed3805d3538ac46cc42f3aadd189a788bd)